dolphinscheduler-dao/src/main/resources/org/apache/dolphinscheduler/dao/mapper/ProcessInstanceMapper.xml

```diff
@@ -368,11 +358,11 @@
 
     <select id="queryByTriggerCode" resultType="org.apache.dolphinscheduler.dao.entity.ProcessInstance">
         select
-        <include refid="baseSqlV2">
-            <property name="alias" value="b"/>
-        </include>
-        from t_ds_trigger_relation a
-        join   t_ds_process_instance b on a.job_id  = b.id
-        where a.trigger_type = 0 and a.trigger_code = #{triggerCode}
+        <include refid="baseSql"/>
+        from t_ds_process_instance a
+        join
+            (select trigger_type, job_id, trigger_code from t_ds_trigger_relation) b
```

Comment:
why do we use sub query here? can we use 
```sql
join t_ds_trigger_relaltion b
```

Reply:
no, there are some columns with the same names in both tables. It would cause ambiguity.

Reply:
so can we add some comment for this line?

Reply:
done

Reply:
PTAL when available : ) @zhongjiajie